cmd/empire/factories.go

```diff
@@ -177,6 +184,27 @@ func newKinesisLogsStreamer(c *cli.Context) (empire.LogsStreamer, error) {
 	return empire.NewKinesisLogsStreamer(), nil
 }
 
+// Events ==============================
+
+func newEventStream(c *cli.Context) (empire.EventStream, error) {
```

Comment:
Would things like this be easier to test, etc, if the functions took the arguments they actually used?  Like in this case it seems kind of wasteful to pass the entire cli.Context in, when all we really care about is a single variable - it also makes it harder to read, since I'm not actually sure, looking at this, what it needs as it hands the cli context further down the stack.